static/lang/re-en.json

```diff
@@ -1101,6 +1104,11 @@
                     "DamageType": "Animated Weapon - Damage Type"
                 }
             },
+            "Werecreature": {
+                "Human": "Human",
+                "Moose": "Moose",
+                "Shark": "Shark"
+            },
```

Comment:
These should be under PF2E.NPCAbility.ChangeShape.Form

Reply:
Good call.